iree/compiler/Conversion/LinalgToSPIRV/test/memref_vecrotization.mlir

```diff
@@ -0,0 +1,37 @@
+// RUN: iree-opt -split-input-file -iree-spirv-vectorize-memref -canonicalize %s | IreeFileCheck %s
+
+// CHECK-LABEL: func @copy
```

Comment:
Nit: It is more readable to 
1) group all the `CHECK-*` commands together
2) align all the `:`
3) Indent the checked lines as it would appear with `dump()`.

So 

```
CHECK-LABEL: func @copy 
 CHECK-SAME:   %[[....]]:
      CHECK:   %[[ALLOC:.+]] = ...
      CHECK:   %[[V:.+]] = ...
...
```